generative/maisi/maisi_inference_tutorial.ipynb

```diff
@@ -0,0 +1,288 @@
+{
```

Comment:
<p>My output from the last cell is:</p>

<pre class="ql-syntax" spellcheck="false">The generated image/mask pairs will be saved in ./output.
controllable_anatomy_size: [['hepatic tumor', 0.3], ['liver', 0.5]]
---- Start preparing masks... ----

  0%|          | 0/1000 [00:00&lt;?, ?it/s]Plan failed with a cudnnException: CUDNN_BACKEND_EXECUTION_PLAN_DESCRIPTOR: cudnnFinalize Descriptor Failed cudnn_status: CUDNN_STATUS_NOT_SUPPORTED (Triggered internally at ../aten/src/ATen/native/cudnn/Conv_v8.cpp:919.)
100%|██████████| 1000/1000 [01:01&lt;00:00, 16.27it/s]

target_tumor_label for postprocess: 26
Current model does not support hepatic vessel by size control, so we treat generated hepatic vessel as part of liver for better visiaulization.
Resampling mask to target shape and spacing
Resize Spacing: [tensor(1.5000, dtype=torch.float64), tensor(1.5000, dtype=torch.float64), tensor(1.5000, dtype=torch.float64)] -&gt; [1.5, 1.5, 2.0]
Output size: [256, 256, 256] -&gt; [256, 256, 256]
---- Mask preparation time: 76.60461235046387 seconds ----
---- Start generating latent features... ----

100%|█████████████████████████████████████████████████████████████████████| 1000/1000 [01:29&lt;00:00, 11.15it/s]
Plan failed with a cudnnException: CUDNN_BACKEND_EXECUTION_PLAN_DESCRIPTOR: cudnnFinalize Descriptor Failed cudnn_status: CUDNN_STATUS_NOT_SUPPORTED (Triggered internally at ../aten/src/ATen/native/cudnn/Conv_v8.cpp:919.)

---- Latent features generation time: 89.74028301239014 seconds ----
---- Start decoding latent features into images... ----
---- Image decoding time: 11.413061380386353 seconds ----
2024-07-08 13:50:01,294 INFO image_writer.py:197 - writing: output/sample_20240708_135001_286914_image.nii.gz
2024-07-08 13:50:02,015 INFO image_writer.py:197 - writing: output/sample_20240708_135001_286914_label.nii.gz
MAISI image/mask generation finished
</pre><p> </p><p>There shows a cudnnException, does this expected or the running env has some requirements?</p>

<pre class="ql-syntax" spellcheck="false">Plan failed with a cudnnException: CUDNN_BACKEND_EXECUTION_PLAN_DESCRIPTOR: cudnnFinalize Descriptor Failed cudnn_status: CUDNN_STATUS_NOT_SUPPORTED (Triggered internally at ../aten/src/ATen/native/cudnn/Conv_v8.cpp:919.)
</pre><p> </p><p>My torch is 2.3.0 and cuda 12.4. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1741/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
I did not see it on my side...

Reply:
<p>I also see "UserWarning: Plan failed with a cudnnException: CUDNN_BACKEND_EXECUTION_PLAN_DESCRIPTOR: cudnnFinalize Descriptor Failed cudnn_status: CUDNN_STATUS_NOT_SUPPORTED (Triggered internally at ../aten/src/ATen/native/cudnn/Conv_v8.cpp:919.)" in your output, it's due to the mismatch cuda version after xformers installed.</p>